lindris.com.domain-authentication.json

```diff
@@ -0,0 +1,56 @@
+{
+  "providerId": "lindris.com",
+  "providerName": "Lindris",
+  "serviceId": "domain-authentication",
+  "serviceName": "Lindris Email Domain Authentication",
+  "version": 1,
+  "logoUrl": "https://app.lindris.com/images/logo.png",
+  "description": "Configures DNS records for domain verification and email authentication to send emails through Lindris",
+  "variableDescription": "mail_server is the target Lindris mail server (e.g., m1.lindris.net, m2.lindris.net, eu.lindris.net); dkim_selector is the DKIM key selector (e.g., key1 for shared keys, or unique identifier for dedicated keys); verification_id and verification_hash are unique credentials for domain ownership verification.",
+  "syncPubKeyDomain": "lindris.com",
+  "syncRedirectDomain": "app.lindris.com",
+  "warnPhishing": false,
+  "records": [
+    {
+      "groupId": "email",
+      "type": "CNAME",
+      "host": "%host%",
```

Comment:
You don't use %host% variable directly. It is added to any record automatically by the protocol.
Put just '@' and it will work. Try in the online editor.